Src/BootCamp.Chapter/MacFactory.cs

```diff
@@ -5,11 +5,64 @@
 
 namespace BootCamp.Chapter
 {
-    public class MacFactory
+    public class MacFactory : ComputerFactory
     {
-        public DesktopComputer Assemble()
+        public override DesktopComputer Assemble()
         {
-            return new DesktopComputer();
+            var computer = new DesktopComputer();
+            TakeBody(computer);
+            PlaceMotherBoard(computer);
+            PlaceHarddisk(computer);
+            PlaceCpu(computer);
+            PlaceGpu(computer);
+            PlaceRam(computer);
+            Console.WriteLine("the mac computer is ready to be shipped");
+            return computer;
+        }
+
+        public override void PlaceRam(DesktopComputer computer)
+        {
+            Console.WriteLine("Im going to place a mac ram");
+            var ram = new Ram("a  mac cpu");
+            computer.SetRam(ram);
+        }
+
+        public override void PlaceGpu(DesktopComputer computer)
+        {
+            Console.WriteLine("Im going to place a mac gpu");
+            var gpu = new Gpu("a  mac cpu");
+            computer.SetGpu(gpu);
+        }
+
+        public override void PlaceCpu(DesktopComputer computer)
+        {
+            Console.WriteLine("Im going to place a mac cpu");
+            var cpu = new Cpu("a  mac cpu");
+            computer.SetCpu(cpu);
+        }
+
+        private void PlaceHarddisk(DesktopComputer computer)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'PlaceHarddisk' a static method.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5124353)

Reply:
done